packages/ses/src/ses.d.ts

```diff
@@ -0,0 +1,38 @@
+type safetyPreference = 'safe' | 'unsafe';
+
+type LockdownOpts = {
+  fringeSet?: {
+    add?: Function,
+    has?: Function,
+  },
+  dateTaming?: safetyPreference,
+  errorTaming?: safetyPreference,
+  mathTaming?: safetyPreference,
+  regExpTaming?: safetyPreference,
+
+  [extraOption:string]?: any,
+};
+
+declare var lockdown = (LockdownOpts) => void;
+
+declare var harden = (any) => any;
+
+declare class Compartment {
+  constructor(endowments: Object);
+  evaluate(command: string): any;
+  globalThis: any;
+  import: () => Promise<any>;
+  importNow: () => any;
+  module: Function;
```

Comment:
〃 and returns `Object` (a module namespace object). If we can specify `ModuleNamespaceObject` as a specialization of `Object` such that it and `string` are the only types allowed on the `modules` compartment constructor function, we’d have a bit stronger contract.